pkg/machine/hyperv/volumes.go

```diff
@@ -40,7 +40,19 @@ func removeShares(mc *vmconfigs.MachineConfig) error {
 
 func startShares(mc *vmconfigs.MachineConfig) error {
 	for _, mount := range mc.Mounts {
-		args := []string{"-q", "--", "sudo", "podman"}
+		args := []string{"-q", "--"}
+
+		cleanTarget := filepath.Clean(mount.Target)
```

Comment:
Did you test this? I doubt this can work, you are cleaning the linux target path on windows which replaces `/` with `\` so the path becomes broken on the linux side.